Source/WebCore/dom/Element.h

```diff
@@ -197,7 +197,7 @@ class Element : public ContainerNode {
 
     void scrollBy(const ScrollToOptions&);
     void scrollBy(double x, double y);
-    virtual void scrollTo(const ScrollToOptions&, ScrollClamping = ScrollClamping::Clamped, ScrollSnapPointSelectionMethod = ScrollSnapPointSelectionMethod::Closest);
+    virtual void scrollTo(const ScrollToOptions&, ScrollClamping = ScrollClamping::Clamped, ScrollSnapPointSelectionMethod = ScrollSnapPointSelectionMethod::Closest, ScrollClass scrollClass = ScrollClass::ScrollTo);
```

Comment:
This is explicitly a `scrollTo` so I'm not sure how the parameter makes sense.

Reply:
See above.